lib/ui/channel_buffers.dart

```diff
@@ -0,0 +1,125 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of dart.ui;
+
+/// A saved platform message for a channel with its callback.
+class StoredMessage {
+  /// Default constructor, takes in a [ByteData] that represents the
+  /// payload of the message and a [PlatformMessageResponseCallback]
+  /// that represents the callback that will be called when the message
+  /// is handled.
+  StoredMessage(this._data, this._callback);
+
+  final ByteData _data;
+  final PlatformMessageResponseCallback _callback;
+
+  ByteData get data => _data;
+  PlatformMessageResponseCallback get callback => _callback;
```

Comment:
every public member must have detailed documentation. see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#writing-prompts-for-good-documentation